content/source/docs/enterprise/private/migrate.html.md

```diff
@@ -9,6 +9,24 @@ sidebar_current: "docs-enterprise2-private-installer-migration"
 This document outlines the procedure for migrating from the AMI-based Private Terraform Enterprise (PTFE)
 to the Installer-based PTFE.
 
+## Backup
+
+Before beginning, it's best create an additional backup of your RDS database. This will allow you to rollback the data and continue to use the AMI if necessary.
```

Comment:
best create > best to create

I think I prefer "roll back" for the verb (roll back the data). But since this is a specialized use of roll back, I don't feel strongly.